source/track_thin.f90

```diff
@@ -580,12 +583,6 @@ subroutine thin4d(nthinerr)
       end if
     end if
     meta_nPartTurn = meta_nPartTurn + napx
-#ifdef BOINC
-    ! call boinc_sixtrack_progress(n,numl)
-    call boinc_fraction_done(dble(n)/dble(numl))
-    continue
-    ! call graphic_progress(n,numl)
```

Comment:
could you leave the commented line in the respective point in the `boinc_turn` subroutine?
I think that Eric added it to remember to implement in the future some graphics...

Reply:
`graphic_progress` is not an actual API routine. But I'll add a comment about it in the `boinc_turn routine instead.